src/panels/calendar/ha-panel-calendar.js

```diff
@@ -0,0 +1,222 @@
+import '@polymer/app-layout/app-header-layout/app-header-layout.js';
+import '@polymer/app-layout/app-header/app-header.js';
+import '@polymer/app-layout/app-toolbar/app-toolbar.js';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-dialog/paper-dialog.js';
```

Comment:
Remove all imports and CSS definitions that you don't need: paper-dialog, paper-icon-item, ...